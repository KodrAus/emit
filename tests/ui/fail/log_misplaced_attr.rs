#![feature(stmt_expr_attributes, proc_macro_hygiene)]

fn main() {
    let a = String::from("hello");

    emit::info!("Some text", value: #[emit::as_display] a);
}
