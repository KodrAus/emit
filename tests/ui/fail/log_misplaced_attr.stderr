error: the emit attribute macros can only be placed on the outside of a field-value expression
 --> fail/log_misplaced_attr.rs:6:37
  |
6 |     emit::info!("Some text", value: #[emit::as_display] a);
  |                                     ^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `emit::as_display` (in Nightly builds, run with -Z macro-backtrace for more info)
